package.json

```diff
@@ -26,14 +26,18 @@
     "babel-plugin-transform-strict-mode": "^6.3.13",
     "babel-preset-react": "6.3.13",
     "bluebird": "^3.1.5",
+    "browser-pack": "^6.0.1",
```

Comment:
This is the npm module `browserify` uses internally. `browserify` didn't work out of the box for us, but I tried to resuse what I could from its internals.